src/wasm/wasm-type.cpp

```diff
@@ -518,28 +523,43 @@ std::ostream& operator<<(std::ostream& os, Signature sig) {
   return os;
 }
 
+std::ostream& operator<<(std::ostream& os, Field::PackedType packedType) {
+  switch (packedType) {
+    case Field::PackedType::not_packed:
+      return os << "not_packed";
+    case Field::PackedType::i8:
+      return os << "i8";
+    case Field::PackedType::i16:
+      return os << "i16";
+  }
+  WASM_UNREACHABLE("unexpected type");
+}
```

Comment:
I think it would make more sense to have the logic for printing `"i8"` or `"i16"` in the print function for `Field` rather than implementing this function for `Field::PackedType`. I can't imagine a situation where it would be useful to print a `Field::PackedType` on its own, since it's kind of just an implementation detail of `Field`.

Reply:
Yeah, that's what I did initially, but then I wondered "what if someone wants to debug this for whatever reason by using `toString`, suspecting there's something wrong with packedType". If that ever happens, there is a slight difference is that printing `PackedType` directly can print `not_packed`, while printing a Field will never print `not_packed` by design. I agree that this is quite a long shot, and I really only implemented it because "why not". Going to revert it :)